testapi.pm

```diff
@@ -1261,10 +1265,13 @@ object (in this example) - so it will ensure the console is active,
 but to setup the root shell on this console, the distribution needs
 to run test code.
 
+After the console selection the distribution callback
+C<$distri->console_selected> is called with C<%args>.
+
 =cut
 
 sub select_console {
-    my ($testapi_console) = @_;
+    my ($testapi_console, %args) = @_;
```

Comment:
why a hash? If you make it an array it's completely up to the distribution to treat it as array or as a hash (I love perl's misfeatures :)